### PR TITLE
FF Cloud landing page

### PR DESCRIPTION
**Pull request by Yndira-E**
## Description

Created the landing page for FlowFuse Cloud and added it under the Product menu

## Related Issue(s)

#2188

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by Yndira-E** [comment]
@Hasmin-AC I've added the meta description keys but left them empty, feel free to add what you see fit 😉

File: src/product/cloud.njk [hunk 1]
```diff
@@ -0,0 +1,161 @@
+---
+layout: page
+nohero: true
+meta:
+    title: "FlowFuse Cloud"
+    description:
```
Comment (Hasmin-AC) [inline src/product/cloud.njk:6]:
```suggestion
    description: "Fully hosted service to build, manage, and deploy Node-RED applications without the hassle of setting up your own cloud infrastructure"
```

File: src/product/cloud.njk [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+            </div>
+        </div>
+    </div>
+</div>    
+
+<div class="max-w-screen-lg mx-auto text-center pt-10 pb-6 px-6 text-lg flex-col flex-row items-center justify-center gap-1 hover:no-underline flex-wrap">
+    <!--Why FlowFuse-->
+    <div class="ff-blue-card">
+        <h2 class="font-bold m-auto w-full text-center sm:text-left mb-6">Why use <span class="text-red-600">FlowFuse Cloud</span></h2>
```
Comment (Hasmin-AC) [inline src/product/cloud.njk:95]:
```suggestion
        <h2 class="font-bold m-auto w-full text-center sm:text-left mb-6">Why use <span class="text-red-600">FlowFuse</span></h2>
```

File: src/product/cloud.njk [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+        <div class="w-full sm:grid sm:grid-cols-2 gap-6 sm:gap-8">
+            {% set sections = [
+                {
+                    "title": "Low Code Advantage",
+                    "description": "FlowFuse Cloud harnesses the power of low code programming to accelerate development times and reduce technical debt. By simplifying the application development cycle, our platform enables businesses to launch solutions faster than traditional coding methods allow."
+                },
+                {
+                    "title": "Cloud vs. Self-Hosted",
```
Comment (Hasmin-AC) [inline src/product/cloud.njk:103]:
```suggestion
                    "title": "Open Source",
```

File: src/product/cloud.njk [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+                    "description": "Choosing FlowFuse Cloud over traditional self-hosted environments means never having to worry about hardware limitations or maintenance. Enjoy unlimited scalability and the flexibility to deploy globally, with the reassurance that your data is backed by the highest standards of cloud security and compliance."
```
Comment (Hasmin-AC) [inline src/product/cloud.njk:104]:
```suggestion
                    "description": "Open source is a key aspect of the FlowFuse product strategy, as we believe it reduces adoption barriers and provides increased transparency and security to all our users. By leveraging open source components like Node-RED, the FlowFuse platform can quickly innovate and adapt to your evolving needs"
```

**Review by Hasmin-AC — APPROVED** [review]

File: src/product/cloud.njk [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+
+<div class="max-w-screen-lg mx-auto text-center pt-10 pb-6 px-6 text-lg flex-col flex-row items-center justify-center gap-1 hover:no-underline flex-wrap">
+    <!--Why FlowFuse-->
+    <div class="ff-blue-card">
+        <h2 class="font-bold m-auto w-full text-center sm:text-left mb-6">Why use <span class="text-red-600">FlowFuse</span></h2>
+        <div class="w-full sm:grid sm:grid-cols-2 gap-6 sm:gap-8">
+            {% set sections = [
+                {
+                    "title": "Low Code Advantage",
+                    "description": "FlowFuse Cloud harnesses the power of low code programming to accelerate development times and reduce technical debt. By simplifying the application development cycle, our platform enables businesses to launch solutions faster than traditional coding methods allow."
```
Comment (Hasmin-AC) [inline src/product/cloud.njk:100]:
```suggestion
                    "description": "FlowFuse Cloud harnesses the power of low code programming to accelerate development times and empower everyone in your organization. By simplifying the application development cycle, our platform enables businesses to launch solutions faster than traditional coding methods allow."
```

File: src/product/cloud.njk [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+                    },
+                    {
+                        "svgPath": "components/icons/lock-closed.svg",
+                        "title": "Robust Security",
+                        "description": "Your data security is our top priority. FlowFuse Cloud employs a multi-layer security approach, ensuring protection at every level, including Role-Based Access Control (RBAC) for granular permissions and Single Sign-On (SSO) for secure access. Our SOC 2 Type 1 compliance demonstrates our commitment to upholding rigorous security standards."
+                    },
+                    {
+                        "svgPath": "components/icons/cloud.svg",
+                        "title": "Hassle-Free Hosting ",
+                        "description": "FlowFuse Cloud handles all your hosting needs. Forget the complexities of managing NGINX, Docker, or any underlying infrastructure; we've got it covered. We handle all the technical details and maintenance, ensuring your Node-RED applications are up and running smoothly. This lets you focus your time and efforts on designing, building, and optimizing your applications."
```
Comment (Hasmin-AC) [inline src/product/cloud.njk:63]:
```suggestion
                        "description": "FlowFuse Cloud handles all your hosting needs. Forget the complexities of managing NGINX, Docker, or any underlying infrastructure. We handle all the technical details and maintenance, ensuring your Node-RED applications are up and running smoothly. This lets you focus your time and efforts on designing, building, and optimizing your applications."
```

**Pull request merged**